microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -1,95 +1,158 @@
+| `proxy-auto-detect` | Forces proxy auto-detection. |
+| `proxy-bypass-list` | A list of hosts for which proxy settings are bypassed; the hosts then use direct connections instead. |
+| `proxy-server` | A proxy server that overrides system settings.  This flag only affects HTTP and HTTPS requests. |
+| `remote-allow-origins` | Enables web socket connections from the specified origins only.  The '*' wildcard allows any origin. |
+| `remote-debugging-port` | Enables remote debugging over HTTP on the specified port. |
+| `RendererAppContainer` | Enables Renderer `AppContainer`. |
+| `Restart` | Indicates that Microsoft Edge WebView2 browser process was restarted (such as after a flag change).  Use this flag to ignore the launch when recording the `Launch.Mode2` metric. |
+| `sdsm-state` | The "Super Duper Secure Mode" state.  Valid values: `off`, `basic`, `balanced`, `strict`. |
+| `SharedArrayBuffer` | If enabled, indicates that a `SharedArrayBuffer` thread is present. |
+| `ThirdPartyStoragePartitioning` | Enables partitioning of third-party storage, such as `IndexedDB` or `CacheStorage`, by the top-level site, to reduce fingerprinting. |
+| `unsafely-treat-insecure-origin-as-secure` | Treats given (insecure) origins as secure origins.  Multiple origins can be specified, as a comma-separated list.  For the definition of secure contexts, see [Secure Contexts](https://w3c.github.io/webappsec-secure-contexts/), including the section [Is `origin` potentially trustworthy?](https://w3c.github.io/webappsec-secure-contexts/#is-origin-trustworthy).  Example: `--unsafely-treat-insecure-origin-as-secure=http://a.test,http://b.test` |
+| `use-fake-device-for-media-stream` | Uses a fake device for Media Stream to replace an actual camera and microphone. |
+| `use-fake-ui-for-media-stream` | Bypasses the media stream infobar, by selecting the default device for media streams (such as WebRTC).  Works with `--use-fake-device-for-media-stream`.  Prefer using `--auto-accept-camera-and-microphone-capture` instead, which doesn't interact with screen capture, such as capturing a browser tab. |
+| `use-system-proxy-resolver` | Uses WinHttp to resolve proxies instead of using WebView2's normal proxy resolution logic.  This flag is only supported in Windows. |
+| `user-agent` | A string used to override the default user agent with a custom user agent. |
+| `user-data-migrated` | Indicates that this process is the product of a relaunch following migration of user data. |
+| `V8Maglev` | Enables the Maglev compiler.  This only sets the V8 flag when manually overridden; otherwise, it defers to whatever the V8 default is. |
+
+<!-- todo: for `proxy-bypass-list`: comma delim'd?  Is a space char to the right of comma supported? -->
```

Comment:
for `proxy-bypass-list`: 
* is it comma delim'd? 
* Is a space char to the right of comma supported?

Reply:
Updated description: Specifies a list of hosts for whom we bypass proxy settings and use direct connections. Ignored unless --proxy-server is also specified. Multiple hosts can be supplied as a comma-separated list.

Reply:
@monica-ch - tech review revised desc:

| `proxy-bypass-list` | A list of hosts for which proxy settings are bypassed; the hosts then use direct connections instead.  This flag is ignored unless `--proxy-server` is also specified.  Multiple hosts can be supplied as a comma-separated list. |

resolving